src/Perspex.Base/Data/ValidationMethods.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Perspex.Data
+{
+    [Flags]
+    public enum ValidationMethods
```

Comment:
I actually can't move this since it is part of the `ValidationStatus` abstract class (unless I change some of that, which I might).